src/main/java/org/skife/jdbi/v2/BeanMapper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2004 - 2014 Brian McCallister
+ * Copyright (C) 2004 - 2015 Brian McCallister and others
```

Comment:
@brianm do you mean all license text project-wide, or just the changes in this PR?